smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -200,159 +222,296 @@ public Task Install()
             agentImage.ForEach(
                 image =>
                 {
-                    Console.WriteLine($"Setting up iotedged with agent image {image}");
+                    Console.WriteLine($"Setting up aziot-edged with agent image {image}");
                 },
                 () =>
                 {
-                    Console.WriteLine("Setting up iotedged with agent image 1.0");
+                    Console.WriteLine("Setting up aziot-edged with agent image 1.0");
                 });
 
-            const string YamlPath = "/etc/iotedge/config.yaml";
-            Task<string> text = File.ReadAllTextAsync(YamlPath);
-            var doc = new YamlDocument(await text);
+            const string KEYD = "/etc/aziot/keyd/config.toml";
+            const string CERTD = "/etc/aziot/certd/config.toml";
+            const string IDENTITYD = "/etc/aziot/identityd/config.toml";
```

Comment:
I'm not seeing these `config.toml` in this PR. I'm assuming it's a hard-coded template that's generated upon aziot-edge installation?